po/zh.po

```diff
@@ -75,7 +75,7 @@ msgstr "压缩缓存"
 
 #: newsboat.cpp:73
 msgid "<command>..."
-msgstr "<命令> ..."
+msgstr "<命令>......"
```

Comment:
I reckon some global replacement has gone wild here, ellipsis should only be three periods. Or does Chinese really use six?

Reply:
Yes, Chinese uses `......` as the ellipsis. Besides, there's indeed a problem about ellipsis.
https://github.com/newsboat/newsboat/blob/c882fcfa721da02b17a19d3f5d173729f3ec42bc/po/zh.po#L78
This phrase is displayed by `newsboat --help` and not treated as a complete sentences. I initially replaced all of the ellipsis to Chinese ones, but it seems too long and not consistent with its surroundings (typically some command line arguments).
Another example which may be more comprehensive:
https://github.com/newsboat/newsboat/blob/c882fcfa721da02b17a19d3f5d173729f3ec42bc/po/zh.po#L25
I decided to act like this: for command line arguments, use `...` to keep it simple and consistent, for others, use `......` instead, since its truly a Chinese sentence. I replaced some of them manually, but this one remains.

Reply:
Wow, interesting! Quite fascinating to see what other parts of the world do. :-)